### ISSUE TITLE
Bug in download And when playing music it stops

### ISSUE THREAD

**Issue by ateprojects** [Arturo254/InnerTune#144]
When I play music it stops or doesn't play anymore and I want to download and it takes too long

**Comment by Arturo254** [comment]
download this version:
https://github.com/Arturo254/OpenTune/releases/download/1.8.0/app-foss-release.apk

**Comment by ateprojects** [comment]
I have that version installed

**Comment by Arturo254** [comment]
the fix version?

**Comment by ateprojects** [comment]
That's right, it doesn't download music that weighs 15 minutes or more, it
takes a long time, I even waited 1 hour and it never downloaded.

**Comment by Arturo254** [comment]
Yes, download version 1.8.0 fix (it is the latest version)
